src/main/java/io/github/thebusybiscuit/slimefun4/core/services/github/ContributionsConnector.java

```diff
@@ -28,6 +28,7 @@
         aliases.put("WalshyDev", "HumanRightsAct");
         aliases.put("J3fftw1", "_lagpc_");
         aliases.put("ajan-12", "ajan_12");
+        aliases.put("LinoxGH", "ajan_12");
```

Comment:
This is redundant since your alias for the GitHub account is already present here:
https://github.com/Slimefun/Slimefun4/blob/master/src/main/java/io/github/thebusybiscuit/slimefun4/core/services/localization/Translators.java#L153

Reply:
Does it make the plugin use ajan_12 head on LinoxGH on Contributors page?

Reply:
Well it should since the code for aliases is the same.